src/devtools/client/debugger/src/components/Editor/ToggleWidgetButton.tsx

```diff
@@ -141,82 +135,84 @@ function QuickActions({
   } else if (hitPoints && isMetaActive) {
     button = <ContinueToNext showNag={showNag} onClick={onContinueToNext} disabled={!next} />;
   } else {
-    button = <AddLogpoint breakpoint={breakpoint} showNag={showNag} onClick={onAddLogpoint} />;
+    button = (
+      <AddLogpoint
+        breakpoint={breakpoint}
+        showNag={showNag}
+        hitPoints={hitPoints}
+        onClick={onAddLogpoint}
+      />
+    );
   }
 
   return (
     <div
+      id="add-print-statement"
       className={classNames(
         "line-action-button absolute z-50 flex translate-x-full transform flex-row space-x-px",
-        enableLargeText && "bottom-0.5"
+        enableLargeText && "bottom-0.5",
+        !hitPoints && "no-hits"
       )}
-      // This is necessary so that we don't move the CodeMirror cursor while clicking.
-      onMouseDown={onMouseDown}
```

Comment:
This is no longer needed from what I can tell since we're switching the editor to `readOnly: "nocursor"` down below.